aviary/models/small_single_aisle/small_single_aisle_GwGm.dat

```diff
@@ -205,7 +205,9 @@
  OFEM=.78,	! off design specified mission Mach number
  ROSCAB=9999.,	! cabin rate of sink during descent (500.) fpm
  RSMX=900.,	! maximum allowable rate of sink during landing approach ft per min (1000)
- FRESF=-.125,     ! Reserve Fuel fraction of mission fuel (minus sign for code trigger)
+ FRESF=0,   ! Reserve Fuel
+ FRESF2=-.125,     ! Reserve Fuel fraction of mission fuel (minus sign for code trigger)
+ FRESF3=2,  ! Use Reserve Fuel
```

Comment:
I'm not up on my GASP so I'm a little confused, why this change?

Reply:
In GASP, FRESF is used for actual total fuel or fraction of total fuel depending on its value. Now, we want FRESF to be read into the total fuel (Aircraft.Design.RESERVES), FRESF2 for fraction of fuel (Aircraft.Design.RESERVES_FRACTION). The third one (FRESF3) is to be read into Aircraft.Design.RESERVES_OPTION.